### DISCUSSION TITLE
Issues with ping

### DISCUSSION THREAD

**Discussion by Rehan587** [termux/termux-app#2548] [Q&A]
I am trying to use ping command in termux.
Here's the output with Termux-info
Application version:
0.117
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages-cf.termux.org/apt/termux-main/ stable main
# sources.list.d/termuxblack.list
deb https://hax4us.github.io/TermuxBlack/ termuxblack main
# science-repo (sources.list.d/science.list)
deb https://mirrors.bfsu.edu.cn/termux/science-packages-24/ science stable
# root-repo (sources.list.d/root.list)
deb https://packages.termux.org/apt/termux-root root stable
# x11-repo (sources.list.d/x11.list)
deb https://packages.termux.org/apt/termux-x11 x11 main
Updatable packages:
All packages up to date
Android version:
11
Kernel build information:
Linux localhost 4.14.186-g4f65b02ee-dirty #1 SMP PREEMPT Thu Dec 16 18:28:32 CST 2021 aarch64 Android
Device manufacturer:
vivo
Device model:
vivo 1915

- the issue starts when I try to ping the websites by its name, but when I ping IPs, it works perfectly fine. For example:-
If I try ping 127.0.0.1, it works perfectly fine. But when I ping www.google.com, it doesn't seem to work, it just freezes after a single ping. 
Here's a screenshot for reference.
![Screenshot_2022_0128_115207.jpg](https://user-images.githubusercontent.com/84271835/151497808-bb56208a-a497-476a-98ef-d17a36a0b7bf.jpg)

**Comment by SDRausty** [comment]
`command -v ping` output please.  If it is in $PREFIX install ping in your proot.  Try `ping 1.1.1.1` to resolve this issue too.

**Reply by Rehan587** [reply]
command -v ping
/data/data/com.termux/files/usr/bin/ping
This is what I got in the output. And for ping 1.1.1.1, it's still stuck

**Reply by SDRausty** [reply]
You are using the Termux `ping` command.  This is might be it is failing.

Install the other `ping` instead.  The one that belongs to the distribution being used in proot.

**Reply by Rehan587** [reply]
I'm closing the issue as I found its solution.. i managed to figure out that my DNS was banned so I used a VPN and it worked.
Closing this issue here.
